README.md

```diff
@@ -164,7 +164,7 @@ E.g. to apply a particular patch to a particular, known-good commit from the
 
 And to build the kernel run:
 
-    $ skt --rc <SKTRC> --state --workdir skt-workdir -vv \
+    $ skt --rc <SKTRC> --state --workdir <WORKDIR> -vv \
```

Comment:
This doesn't look like the previous version.

Reply:
Ah, right! I'm still getting confused by GitHub's interface (versus very distinct color scheme I use for diffs) so I missed the dividing light blue line and thought this belongs to the above specific example. Ignore this comment then.